### DISCUSSION TITLE
How can I do a simple parse of a multi-document file?

### DISCUSSION THREAD

**Discussion by ind1go** [eemeli/yaml#588] [Q&A] [answered]
I've got a multi-document file, with the documents separated by `...`. What I'd _like_ to do is to a simple `parse`, to get simple objects representing only the data structure, that I can assert as my own TypeScript type.

However, because it's multi-document, I have to use `parseAllDocuments()` (due to `YAMLParseError: Source contains multiple documents; please use YAML.parseAllDocuments()`), which gives me the much more complex structure with all the metadata.

Is there a way to get from the more complex structure produced by `parseAllDocuments()` to an array of the type of structure produced by `parse`?

**Comment by eemeli** [comment, marked as answer]
```js
parseAllDocuments(...).map(doc => doc.toJS())
```
